.github/workflows/snap.yml

```diff
@@ -102,6 +102,7 @@ jobs:
             ~/.cache/snapcraft/log/
             ~/.local/state/snapcraft/log/
             ${{ env.SNAP_NAME }}_*.txt
+            snapcraft-${{ env.SNAP_NAME }}-*.txt
```

Comment:
Do we also need this on the local build? I had a bit of a search but can't find what code writes to this pattern (but from your log extract, something clearly does). The pattern in the line above can be seen in `fetch_logs()` in craft-application. Maybe something in the remote build adds the `snapcraft-` prefix? Maybe this is the correct name and the previous one is wrong? (It seems odd that both could be the case).